lib/executor/ramping_vus_test.go

```diff
@@ -36,6 +37,26 @@ func TestRampingVUsConfigValidation(t *testing.T) {
 	errs = c.Validate()
 	require.NotEmpty(t, errs)
 	assert.Contains(t, errs[0].Error(), "greater than 0")
+
+	c.StartVUs = null.IntFrom(100000001)
+	errs = c.Validate()
+	require.NotEmpty(t, errs)
+	assert.Contains(t, errs[0].Error(), "number of VU shifts is too large")
+
+	c.StartVUs = null.IntFrom(60000000)
+	c.Stages = []Stage{
+		{Target: null.IntFrom(50000000), Duration: types.NullDurationFrom(12 * time.Second)},
+		{Target: null.IntFrom(50000000), Duration: types.NullDurationFrom(12 * time.Second)},
+	}
```

Comment:
As I reviewed the test, I wondered, "why those specific values?". Thus, I would suggest to either rely on the constant I suggested before, in a way that makes the reason why we pick a number explicit: `MaxRampingVUShift + x`, or to at least add a comment explaining why we use that value 🙇🏻